skopt/dummy_opt.py

```diff
@@ -28,7 +30,23 @@ def dummy_minimize(func, dimensions, n_calls=100, random_state=None):
           `Categorical`).
 
     * `n_calls` [int, default=100]:
-        Number of calls to `func` to find the minimum.
+        Maximum number of calls to `func` to find the minimum.
```

Comment:
For now, I think we should just retain the previous better definition of better `n_calls`, till we have better stopping criteria. See https://github.com/scikit-optimize/scikit-optimize/issues/6